Event handlers

mikethreels opened the pull request:
## Milestone: Calculations

### When updating the project the following points have been added.
- [x] Changed App to a stateful component.
- [x] implemented the handleClick function.
- [x] implemented the onClick event.

apologies for the lack of commits.

mikethreels committed:
feat: connected calculate to app component

mikethreels committed:
feat: updated readme file

kubilaycaglayan commented on src/components/button_panel.jsx:
```diff
@@ -0,0 +1,45 @@
+        <Button key="÷" name="buttongroup1last" clickHandler={handleClick} buttonName="÷" />
+      </div>
+      <div className="group2">
+        <Button key="8" name="buttongroup2" clickHandler={handleClick} buttonName="8" />
+        <Button key="9" name="buttongroup2" clickHandler={handleClick} buttonName="9" />
+        <Button key="7" name="buttongroup2" clickHandler={handleClick} buttonName="7" />
```
Let's correct the number order. 😊

kubilaycaglayan reviewed (changes requested):
#### Hello Team 👋
#### Status: Changes Required ♻️

Great job so far! 👏
Click handling is great 👍
Let's cover a few details together to make your project better 😊⏬

### Button Alignment

- [x] On the top row of number buttons, please fix the order of the numbers. Currently, it goes like 8, 9, 7.

### Behaviour

- [x] 160 / 2 = and then % throws a prop type warning on the console.

![image](https://user-images.githubusercontent.com/60448833/99199118-ef25f600-27ad-11eb-9b2c-19d0413edbde.png)

- [x] Consider the edge case, add error handling when the user tries to divide a number by zero. Currently, the application is breaking. 12 / 0 =

## That's it, you can submit another code review request after making these changes ♻️😇

With all my respect 🙏
> Kubilay Caglayan | Technical Support Engineer

mikethreels committed:
fix: fixed the button order, the type error and divided by 0 as suggested by the TSE

mikethreels commented:
## Milestone: Calculations Update

### When updating the project the following points have been added.
- [x] fixed the order of the buttons.
- [x] fixed the type error when using the % button.
- [x] covered the edge case when divided by 0.
- [x] make sure the divided by 0 error is cleared.

mikethreels committed:
fix: cleared the div by 0 error when making another calculation

Torres-ssf reviewed (approved):
### Milestone Approved :+1: :clap: 
Hi @mikethreels :hand: 
Awesome job implementing the requested changes :clap: 
Good luck with the next project.

Happy Coding :computer: 
**Sergio Torres**
